pkg/tfbridge/detailed_diff.go

```diff
@@ -480,6 +481,13 @@ func MakeDetailedDiffV2(
 	ps map[string]*SchemaInfo,
 	priorProps, props, newInputs resource.PropertyMap,
 ) map[string]*pulumirpc.PropertyDiff {
+	stripSecretsAndOutputs := func(props resource.PropertyMap) resource.PropertyMap {
```

Comment:
You can add a comment here that the safety of doing this hinges on the fact that we return indices to the engine, not values, and the engine takes care of masking the values.